### PR TITLE
Enabled mandatory run update-props as part of make targets

### PR DESCRIPTION
**Pull request by PinGwynn**
All currents (already templated) properties files moved as examples.
Added run update-props as a dependency of "run-dev" and "up-test-mode" make targets.

For now, only one place for management properties files is templates. All local changes will be overwritten as part of update-props.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,12 @@ jobs:
         - make update-props-dryrun
     - stage: unit
       language: java
+      before_install:
+        - sudo pip install --upgrade pip
+        - sudo pip install ansible
+      #  - sudo apt-get update
```
Comment (abochkarev) [inline .travis.yml]: Just to clarify: do we really need to have commented out commands in the travis.yml?

Comment (PinGwynn) [reply .travis.yml]: Thank you. Comments have been removed

File: base/hacks/load_kilda_dev_ubuntu.sh [hunk 2]
```diff
@@ -85,7 +85,7 @@ echo ##  PYTHON ETC
 echo ##########################
 
 sudo apt-get update
-sudo apt-get install -y apt-utils python python-pip
```
Comment (surabujin) [inline base/hacks/load_kilda_dev_ubuntu.sh]: Where is it(this whole file) used?

Comment (PinGwynn) [reply base/hacks/load_kilda_dev_ubuntu.sh]: This file uses for prepare local dev environment. It installs all necessary stuff, like java, python, compose, etc...

Comment (surabujin) [reply base/hacks/load_kilda_dev_ubuntu.sh]: i.e. it should be started manually to prepare local env(linux ubuntu) to build open-kilda?

Can't find its mentions in docs/wiki (actually I can't find its mentions anywhere).

**Pull request merged**